src/lib/questions.ts

```diff
@@ -504,6 +595,8 @@ export const questionsAndText: (
 				message: "Do you prefer double or single quotes?",
 				initial: "double",
 				choices: ["double", "single"],
+				import: async (ctx) =>
+					(await ctx.analyzeCode('"', "'")) ? "double" : "single",
```

Comment:
If ESLint is used, we could look at the quotes rule instead.

Reply:
See above.